lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -119,16 +118,16 @@
                     var $submissionContainer = $('.submission-error'),
                         $errorMessageContainer = $submissionContainer.find('.message-copy'),
                         $countryLabel = $('#user-country-title'),
-                        txt = [
-                            'Please go to your {link_start}profile page{link_end} ',
-                            'and provide your country of residence.'
-                        ],
-                        msg = window.interpolate_text(
-                            // Translators: link_start and link_end denote the html to link back to the profile page.
-                            gettext(txt.join('')),
+                        msg = HtmlUtils.interpolateHtml(
+                            // Translators: link_start and link_end
+                            // denote the html to link back to the
+                            // profile page.
+                            gettext('Please go to your {link_start}profile page{link_end} and provide your country of residence.'), // jshint ignore:line
                             {
-                                link_start: '<a href="' + this.context.account_settings_url + '">',
-                                link_end: '</a>'
+                                link_start: HtmlUtils.HTML(
```

Comment:
Use ``HtmlUtils.joinHtml`` instead of concatenating strings (or you can use ``HtmlUtils.interpolate`` if you prefer).